admin/python/lsst/qserv/admin/nodeMgmt.py

```diff
@@ -123,11 +123,11 @@ def selectDict(self, state=None, nodeType=None):
         """
 
         # if state is a string make a list out of it
-        if isinstance(state, types.StringTypes):
+        if isinstance(state, str):
```

Comment:
Yes, I had not meant to leave that change in. Sorry. This needs to be dealt properly for python portability (including the option of jut going to python 3).